src/kuberesources.ts

```diff
@@ -19,6 +19,7 @@ export const allKinds = {
     job: new ResourceKind("Job", "Jobs", "Job", "job"),
     cronjob: new ResourceKind("CronJob", "CronJobs", "CronJob", "cronjob"),
     pod: new ResourceKind("Pod", "Pods", "Pod", "pod"),
+    crd: new ResourceKind("CRD", "CRDs", "CRD", "crd"),
```

Comment:
Don't have strong feelings about this but wondered if we should spell it out in the display strings - `Custom Resource/s` or `Custom Resource Definition/s`.  Thoughts?

Reply:
Custom Resource/s seems better, fixed!

Reply:
yup